README.md

```diff
@@ -720,6 +779,15 @@ __AWESOME__ apps counter: __121__ 🎉
 - [x] [GitHub](https://github.com/florisboard/florisboard)
 - [ ] Official page
 
+## Loyalty Card
+### Catima ❤️
```

Comment:
Could you please explain me how does this app work from the practical standpoint?